awx_collection/tests/integration/targets/project/tasks/run_it.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Run Integration Test
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  environment:
+      TOWER_HOST: https://tower-ansible-automation-platform.apps-crc.testing
+      TOWER_USERNAME: admin
+      TOWER_PASSWORD: jWidobmi5rfdw7yensgjM8RRMmkt2Pat
```

Comment:
👀

Reply:
Thats temporary, let me remove it from the PR, I was leaving it in to run testing.